frontend/src/libs/components/icon/icon-name-to-plain-svg.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactComponent as ArrowIcon } from '#assets/icons/arrow.svg';
+import { ReactComponent as AvatarIcon } from '#assets/icons/avatar.svg';
+import { ReactComponent as ChatIcon } from '#assets/icons/chat-logo.svg';
+import { ReactComponent as NotificationIcon } from '#assets/icons/notification.svg';
+import { ReactComponent as PlusIcon } from '#assets/icons/plus.svg';
+import { ReactComponent as SignOutIcon } from '#assets/icons/sign-out.svg';
+import { ReactComponent as SubscriptionIcon } from '#assets/icons/subscription.svg';
+import { type IconName } from '#libs/types/types.js';
+
+const iconNameToPlainSvgMap: Record<
```

Comment:
rename according to this [comment](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/140) and cp-55 pr

Reply:
Which comment? It's the link to the entire PR

Reply:
Oops. Here is the [link](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/140#discussion_r1312696725) Also it's names IconNameToIconin current development branch